### PR TITLE
update to announce impace of File Provider

### PR DESCRIPTION
**Pull request by cartergreen**
@gacarini

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/deploy-and-configure-on-macos.md [hunk 1]
```diff
@@ -230,7 +230,10 @@ This setting specifies whether Files On-Demand is enabled.
 > [!IMPORTANT]
 > We recommend keeping Files On-Demand enabled. [See all our recommendations for configuring the sync app](ideal-state-configuration.md)
 
-If you don't set this setting, Files On-Demand will be enabled automatically as we roll out the feature, and users can turn the setting on or off.
+If you don't set this setting, Files On-Demand will be enabled automatically as we roll out the feature, and users can turn the setting on or off.  
+
+> [!NOTE]
+> Beginning in macOS 12.1 Monterey Files On-Demand will be the default and there will not be a setting for users to turn it off.
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:236]:
```suggestion
> Beginning in macOS Monterey 12.1, Files On-Demand will be the default and there will not be a setting for users to turn it off.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cartergreen, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@adeejoseph 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**